ctypes_test.py

```diff
@@ -0,0 +1,4 @@
+import ctypes
```

Comment:
In the end i will delete it. Only for demonstration

Reply:
You can place it in the tests folder, probably `CtypesUsing.py` would love it, but of course then you should use a DLL that exists everywhere or guard with `os.name` tests, etc.